### ISSUE TITLE
PoisonError on gumdrop_derive

### ISSUE THREAD

**Issue by RazrFalcon** [anderejd/cargo-geiger#21]
Trying to check [usvg/cli](https://github.com/RazrFalcon/usvg/tree/master/cli):

```
    ... 
    Checking lyon_geom v0.10.1
    Checking usvg v0.2.0 (file:///media/data/Programming/Projects/rust/usvg)
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', libcore/result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
   Compiling gumdrop_derive v0.4.1 (https://github.com/murarth/gumdrop#26c7a2f4)
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: "PoisonError { inner: .. }"', libcore/result.rs:945:5
^C
```

**Comment by anderejd** [comment]
Thank you for the report!
I suspect this has the same cause as #19 but let's keep this issue open in case it hasn't.

**Comment by anderejd** [comment]
Fixed by #23
